examples/flutter_gallery/fuchsia/meta/flutter_gallery.cmx

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
how do you generate this file?

Reply:
There's currently no automated way to generate the cmx files. I initially copied one from the Fuchsia tree for the stocks example, then copied that one here, changing the component name.

Reply:
We'll likely need to generate one when we want to support Fuchsia with 'flutter create' though.

Reply:
oh awesome!